<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>243</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QProgressBar" name="pbar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어라운드OTF Bold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_file_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>File_name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_savedir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어라운드OTF Bold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>fnc_stop_progress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_open_savedir</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>fnc_open_savedir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fnc_stop_progress()</slot>
  <slot>fnc_open_savedir()</slot>
 </slots>
</ui>
